css/index.css

```
@@ -1 +1,339 @@
-/* Compile your LESS file! */
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+  font-size: 62.5%;
+}
+.info-top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+.head {
+  display: flex;
+  justify-content: space-between;
+}
+.nav-order {
+  margin: 5px;
+  text-decoration: none;
+}
+.section-end {
+  border-top: 2px solid red;
+  padding-top: 10px;
+}
+.name-address-cascade {
+  display: flex;
+  align-items: baseline;
+  margin-top: 10px;
+}
+.container {
+  width: 1000px;
+  margin: 0 auto;
+  padding: 20px;
+  border: 1px solid black;
+}
+@media (max-width: 500px) {
+  .container {
+    font-family: 'ZCOOL KuaiLe', cursive;
+  }
+}
```

```suggestion
@media (max-width: 500px) {
  .container {
    font-family: 'ZCOOL KuaiLe', cursive;
  }
```
Nice nested media query. Also I like the font choice!